public/Copy-DbaDbCertificate.ps1

```diff
@@ -304,6 +304,20 @@ function Copy-DbaDbCertificate {
                                         $usingtempfiles = $true
                                     } else {
                                         $export = Backup-DbaDbCertificate @params
+
+                                        # The exported files are only readable by the source instance account
+                                        # But for the restore they need to be readable by the targe instance account
+                                        # Current solution is to try to make them readable to everyone and remove them after the restore
```

Comment:
isn't thw whole premise to use a share that has certs in there useful for disaster recovery scenario ? the deletion in here changes decisevely, as "before" it was just in case of errors (catch), while now it's in each and every case (finally)

Reply:
Only newly created cert files that where created by the backup part are deleted. If there are cert files present, then $usingtempfiles is true and the files are not deleted. In our tests we don't have files there so we can only test the Copy- (backup + restore) path.